### PR TITLE
committing testcase2 to branchx

### PR DESCRIPTION
**Pull request by SAGAR-MSD**
Hi Lead,

I have committed to test 2.
Please review and approve. 

Regards,
-Sagar
sagarmsd1397@gmail.com

### CODE REVIEW

**Comment by SAGAR-MSD** [comment]
Please review and approve.

File: src/lead/TestCase2.java [hunk 1]
```diff
@@ -0,0 +1,15 @@
+package lead;
+
+public class TestCase2 
+{
+	public static void main(String[] args) 
+	{
+		String hungry = "Yes";
```
Comment (Sagar-Manjunath) [inline src/lead/TestCase2.java:7]: Add few more checks.

**Comment by Sagar-Manjunath** [comment]
Hi Sagar,
Your code looks good. Please add one more check.
Regards,
-TestLead

**Comment by SAGAR-MSD** [comment]
Hello Lead,

I have added the additional checks and verified them.
Please review and approve.

Regards,
-Sagar

**Comment by Sagar-Manjunath** [comment]
Hi Sagar,

Your code looks good. Please go ahead with merge  to new branch

Regards,
-Sagar

**Pull request merged**